src/loaders/svg/tvgSvgUtil.cpp

```diff
@@ -163,14 +162,21 @@ float svgUtilStrtof(const char *nPtr, char **endPtr)
         unsigned int expo_part;
         int minus_e;
 
-        iter++;
+        ++iter;
+
+        //Exception: svg may have 'em' unit for fonts. ex) 5em, 10.5em
+        if ((*iter == 'm') || (*iter == 'M')) {
```

Comment:
for now any unit indicator present after the number is returned from this funct. 
and after this patch the 'em' unit will be the only one that is "eaten" in this function. 
for consistency I'd rather add to the if-else block below an extra 'else' in which I'll return the "eaten" 'e'.

Reply:
@mgrudzinska sorry i can't catch your point.. so which line this condition supposed to be ?

Reply:
sorry for not being precise
it's l.194 in the original file.
I'd add there:
```
else {
    a = iter - 1;
    goto on_success;
}
```
what do you think about such solution?

Reply:
so if there is anything else than '+', '-" or a digit, we leave it

Reply:
@mgrudzinska  This handles exponential part and it might not going there by  "else if (!isdigit(*(a - 1)))"

        //exponential part
        expo_part = 0;
        if (isdigit(*iter))
        {
            while (*iter == 0) iter++;

            for (; isdigit(*iter); iter++)
            {
                expo_part = expo_part * 10U + (unsigned int)(*iter - '0');
            }
        }
        else if (!isdigit(*(a - 1)))
        {
            a = nPtr;
            goto on_success;
        }
        else if (*iter == 0) goto on_success;


even does,  I think the purpose is different contextually . 

0. Exception Handling
1. signed or not
2. Exponential part

either way, no correct answer. it's trivial by me.

Reply:
I'm not sure any more about the 'eating' that I mentioned before. Sorry.

Reply:
So probably your conception was ok